From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
Subject: [PATCH] Unbundle gems
Date: Mon, 18 Oct 2021 13:22:48 +0200
From: Jakub Jirutka <jakub@jirutka.cz>

Don't install bundled gems - we package them separately.

Origin: Alpine Linux
Upstream-Status: Unknown
[ismael@iodev.co.uk: Normalised patch and edited commit message]
Signed-off-by: Ismael Luceno <ismael@iodev.co.uk>
---
 tool/rbinstall.rb |    2 ++
 1 file changed, 2 insertions(+)

--- a/tool/rbinstall.rb
+++ b/tool/rbinstall.rb
@@ -947,6 +947,7 @@
   end
 end
 
+=begin XXX-Patched
 install?(:ext, :comm, :gem, :'bundled-gems') do
   gem_dir = Gem.default_dir
   install_dir = with_destdir(gem_dir)
@@ -1026,5 +1027,6 @@
     puts "skip installing bundled gems because of lacking zlib"
   end
 end
+=end
 
 parse_args()
 
